extensions/websocket/README.md

```diff
@@ -0,0 +1,436 @@
+# @loopback/websocket
+
+This module allows integrating [socket.io](http://socket.io) to handle
+connections in a friendly way through drivers.
+
+## Stability: ⚠️Experimental⚠️
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Installation
+
+```sh
+npm install --save https://github.com/arondn2/loopback4-extension-websocket
+```
+
+## Example
+https://github.com/arondn2/loopback4-example-websocket
+
+## How to use
+
+### Instancing the application
+
+#### With Basic WebsocketApplication
+Extends the application of `WebsocketApplication` class which has everything
+need it.
+```typescript
+import { WebsocketApplication } from '@loopback/websocket';
+
+export class MyApplication extends WebsocketApplication {
+}
+```
+
+#### With Basic Loopback 4 Application
+Load de component in the application constructor then get websocket server
+instance with `WebsocketBindings.SERVER` binding. Also, overwrite `start` and
+`stop` methods to start and stop websocket server.
+
+```typescript
+import { Application, ApplicationConfig } from '@loopback/core';
+import { WebsocketServer, WebsocketBindings } from '@loopback/websocket';
+
+export class MyApplication extends Application {
+  // Property for ws server
+  public readonly wsServer: WebsocketServer;
+  constructor(options: ApplicationConfig = {}) {
+    super(options);
+    
+    // Load ws component
+    this.component(WebsocketComponent);
+    
+    // Instance ws server
+    this.wsServer = this.getSync(WebsocketBindings.SERVER);
+  }
+  
+  // Start ws server
+  async start() {
+    await this.wsServer.start();
+  }
+  // Stop ws server
+  async stop() {
+    await this.wsServer.stop();
```

Comment:
websocket server is not started by default without these lines?

Reply:
to be honest, i am no sure, but, I could swear they are required. I will make some tests